android/src/main/res/values-tr/strings.xml

```diff
@@ -0,0 +1,207 @@
+    <string name="error_request_token_amount">Belirteç için istenen tutar ayarlanamadı \'%1s\'</string>
+    <string name="error_request_token_but_no_erg">Token talep edildi, ancak ERG miktarı yok. Varsayılan bir tutar gönderilecektir.</string>
+
+    <!-- Cold wallet signing request screen -->
+    <string name="title_signing_request">İmza</string>
+    <string name="desc_signing_request">Lütfen verilecek işlemin ayrıntılarını kontrol edin.</string>
+    <string name="title_inboxes">Harcanan miktarlar</string>
+    <string name="desc_inboxes">Bu tutarlar, işlem işlendiğinde harcanacaktır.</string>
+    <string name="title_outboxes">Giden tutarlar</string>
+    <string name="label_qr_pages_info">%1s den %2s</string>
+    <string name="desc_outboxes">İşlem işlendiğinde bu tutarlar gönderilecektir.</string>
+    <string name="desc_show_signed">İşleminiz imzalandı ve ağa gönderilmesi gerekiyor. Devam etmek için bu QR kodunu tarayın.</string>
+    <string name="desc_show_signed_multiple">İşleminiz imzalandı ve ağa gönderilmesi gerekiyor. Devam etmek için bu QR kodlarını tarayın.</string>
+
+    <!-- Ergo Pay Signing -->
+    <string name="title_ergo_pay_request">Ergo Ödeme Talebi</string>
+    <string name="label_sign_with">İmzala: %1s (%2s)</string>
+    <string name="label_fetching_data">dApp'den veri isteme…</string>
+    <string name="label_message_from_dapp">dApp'den mesaj:\n%1s</string>
+    <string name="label_error_occured">Bir hata oluştu:\n%1s</string>
+    <string name="label_ergo_pay_wrong_address">Bu cüzdana ait olmayan %1s adresi için imza talebi alındı.</string>
+    <string name="label_ergo_pay_choose_address">Lütfen Ergo Ödeme Talebini işlemek için bir adres seçin.</string>
+    <string name="error_prover_cant_sign">İşlem bu cüzdanın anahtarlarıyla imzalanamadı.</string>
+
+    <!-- Settings -->
+    <string name="desc_about">Benjamin Schulte tarafından getirildi, %1s</string>
+    <string name="about_year" translatable="false">2022</string>
+    <string name="desc_about_moreinfo">Bu proje <a href="https://github.com/ergoplatform/ergo-wallet-app/blob/master/LICENSE">Apache 2 License</a> altında lisanslanmıştır. <a href="https://github.com/ergoplatform/ergo-wallet-app">Açık kaynak yazılım</a>\n
+    Yardım için <a href="https://discord.gg/kj7s7nb">Discord</a> veya <a href="https://t.me/ergosupport">Telegram</a></string>
+    <string name="desc_coingecko">Ergo fiyatı <a href="https://www.coingecko.com/en/coins/ergo">CoinGecko</a> tarafından desteklenmektedir.</string>
+    <string name="label_cg_conn_error">Coingecko'ya bağlanırken hata oluştu, lütfen daha sonra tekrar deneyin.</string>
+    <string name="desc_dark_mode">Karanlık veya aydınlık mod?</string>
+    <string name="button_dark_mode_system">Sistem varsayılanı</string>
+    <string name="button_dark_mode_day">Aydınlık</string>
+    <string name="button_dark_mode_night">Karanlık</string>
+    <string name="desc_expert_settings">Uzman Ayarlar</string>
```

Comment:
* Uzmanlar için ayarlar (Settings for experts)
* Gelişmiş ayarlar (Advanced settings)

One of these would be better.